src/pages/FaqPage/FaqPage.jsx

```diff
@@ -0,0 +1,136 @@
+import { useState, useEffect, useCallback } from "react";
+import debounce from "lodash.debounce";
+import useMetaTags from "../../hooks/useMetaTags";
+import styles from "./FaqPage.module.css";
+import { faqData } from "./faqData";
+
+const FaqPage = () => {
+    const [searchTerm, setSearchTerm] = useState("");
+    const [filteredData, setFilteredData] = useState(faqData);
+
+    useMetaTags({
```

Comment:
I'm curious... what are MetaTags for? 😅

Reply:
MetaTags are used in the FAQ page to set metadata like the title, description, and Open Graph tags, which improve SEO, enhance social media sharing, and help search engines understand the page's content.